components/ui-api-layer/internal/domain/k8s/resourcequota_resolver.go

```diff
@@ -30,3 +71,153 @@ func (r *resourceQuotaResolver) ResourceQuotasQuery(ctx context.Context, environ
 
 	return r.converter.ToGQLs(items), nil
 }
+
+func (r *resourceQuotaResolver) ResourceQuotaStatus(ctx context.Context, environment string) (gqlschema.ResourceQuotaStatus, error) {
```

Comment:
there is a business logic here, but the CONTRIBUTING.md defines:
```
Keep the first level of a domain package consistent. Create these files for every resource:
    - {NAME}_resolver.go, which contains resolver type. They usually call services and convert types.
    - {NAME}_service.go, which contains the business logic. The service uses data transfer object (DTO) type.```

The resolver methods should be simple, call the service with business logic, log error (if occurred), return new error, which does not provide details (will be shown in the response), call converter (if needed) - see ResourceQuotasQuery